### PR TITLE
Add avg and std calculation for time and throughput

### PR DESCRIPTION
**Pull request by quasiben**
PR adds AVG/STD calc for time and throughput

```
===============================
Wall-clock     | Throughput
-------------------------------
294.73 ms      | 82.83 MiB/s
220.04 ms      | 110.95 MiB/s
247.50 ms      | 98.64 MiB/s
214.36 ms      | 113.89 MiB/s
224.59 ms      | 108.71 MiB/s
231.22 ms      | 105.59 MiB/s
228.30 ms      | 106.94 MiB/s
296.44 ms      | 82.36 MiB/s
226.30 ms      | 107.89 MiB/s
221.86 ms      | 110.04 MiB/s
===============================
Throughput | 102.78 MiB +/- 10.78 MiB
Wall-Clock | 240.53 ms +/- 28.74 ms
```

### CODE REVIEW

**Review by galipremsagar — APPROVED** [review]

File: dask_cuda/benchmarks/local_cudf_merge.py [hunk 1]
```diff
@@ -291,12 +291,21 @@ def main(args):
...
+        t_p.append(throughput)
         m += " " * (15 - len(m))
         print(f"{m}| {format_bytes(throughput)}/s")
         t_runs[idx] = float(format_bytes(throughput).split(" ")[0])
+    t_p = numpy.asarray(t_p)
+    times = numpy.asarray(times)
+    print("===============================")
+    print(f"Throughput | {format_bytes(t_p.mean())} +/- {format_bytes(t_p.std()) }")
+    print(f"Wall-Clock | {format_time(times.mean())} +/- {format_time(times.std()) }")
     print("===============================")
```
Comment (pentschev) [inline dask_cuda/benchmarks/local_cudf_merge.py:309]: Sorry for the nitpicking, but could we improve alignment? It seems like it's all misaligned from the example you posted.

File: dask_cuda/benchmarks/local_cudf_merge.py [hunk 2]
```diff
@@ -291,12 +291,21 @@ def main(args):
...
+        times.append(took)
+        t_p.append(throughput)
         m += " " * (15 - len(m))
         print(f"{m}| {format_bytes(throughput)}/s")
         t_runs[idx] = float(format_bytes(throughput).split(" ")[0])
+    t_p = numpy.asarray(t_p)
+    times = numpy.asarray(times)
+    print("===============================")
+    print(f"Throughput | {format_bytes(t_p.mean())} +/- {format_bytes(t_p.std()) }")
+    print(f"Wall-Clock | {format_time(times.mean())} +/- {format_time(times.std()) }")
```
Comment (pentschev) [inline dask_cuda/benchmarks/local_cudf_merge.py:308]:
```suggestion
    print(f"Throughput     | {format_bytes(t_p.mean())} +/- {format_bytes(t_p.std()) }")
    print(f"Wall-Clock     | {format_time(times.mean())} +/- {format_time(times.std()) }")
```

File: dask_cuda/benchmarks/local_cudf_merge.py [hunk 3]
```diff
@@ -291,12 +291,21 @@ def main(args):
     print("===============================")
```
Comment (pentschev) [inline dask_cuda/benchmarks/local_cudf_merge.py:291]:
```suggestion
    print("================================================================================")
```

File: dask_cuda/benchmarks/local_cudf_merge.py (hunk 1 above)
Comment (pentschev) [inline dask_cuda/benchmarks/local_cudf_merge.py:309]:
```suggestion
    print("================================================================================")
```

File: dask_cuda/benchmarks/local_cudf_merge.py [hunk 4]
```diff
@@ -291,12 +291,21 @@ def main(args):
...
     print("Wall-clock     | Throughput")
     print("-------------------------------")
```
Comment (pentschev) [inline dask_cuda/benchmarks/local_cudf_merge.py:293]:
```suggestion
    print("--------------------------------------------------------------------------------")
```

**Review by pentschev — COMMENTED** [review]
Perhaps these changes should do it.

File: dask_cuda/benchmarks/local_cudf_merge.py (hunk 1 above)
Comment (quasiben) [reply dask_cuda/benchmarks/local_cudf_merge.py:309]: sure!

**Comment by quasiben** [comment]
New changes look like the following:

```
===============================
Wall-clock     | Throughput
-------------------------------
304.65 ms      | 80.14 MiB/s
226.29 ms      | 107.89 MiB/s
227.56 ms      | 107.28 MiB/s
221.32 ms      | 110.31 MiB/s
229.52 ms      | 106.37 MiB/s
234.61 ms      | 104.06 MiB/s
228.25 ms      | 106.96 MiB/s
225.67 ms      | 108.19 MiB/s
214.79 ms      | 113.67 MiB/s
221.88 ms      | 110.03 MiB/s
===============================
Throughput | 105.49 MiB +/- 8.80 MiB
Wall-Clock | 233.45 ms +/- 24.27 ms
===============================
```

**Comment by pentschev** [comment]
Did you paste the right one in https://github.com/rapidsai/dask-cuda/pull/828#issuecomment-1011203989 @quasiben ? Seems exactly the same as before.

File: dask_cuda/benchmarks/local_cudf_merge.py (hunk 3 above)
Comment (pentschev) [reply dask_cuda/benchmarks/local_cudf_merge.py:291]: Did you miss this one @quasiben ?

File: dask_cuda/benchmarks/local_cudf_merge.py [hunk 5]
```diff
@@ -290,14 +290,25 @@ def main(args):
     print(f"data-processed | {format_bytes(took_list[0][0])}")
     print("===============================")
     print("Wall-clock     | Throughput")
-    print("-------------------------------")
+    print("-" * 80)
```
Comment (pentschev) [inline dask_cuda/benchmarks/local_cudf_merge.py:293]: Nice, you're much smarter than me! 😄

**Comment by quasiben** [comment]
@pentschev I think i've got all the issues resolved.  When you have time can you take another look ?

**Review by pentschev — APPROVED** [review]
LGTM, thanks @quasiben !

**Comment by pentschev** [comment]
@gpucibot merge

**Comment by pentschev** [comment]
@gpucibot merge

**Pull request merged**